certified-connectors/TrueDialog SMS/readme.md

```diff
@@ -0,0 +1,52 @@
+
+## The TrueDialog Connector 
+
+TrueDialog's SMS API is enterprise-grade with more pre-built features than any other SMS software. Built for developers and hosted in the Microsoft Azure cloud, our SMS API is highly reliable, secure and includes a pre-built ISO 9001 database for extra opt-out and compliance protection. Also available through our full-featured Enterprise-Grade SMS Texting Software.
+
+[Learn more](https://www.truedialog.com/sms-api-texting-api/) about what sets TrueDialog’s API apart from the rest.
+
```

Comment:
@TrueDialogDynamics Please go thru the [template provided for readme.md](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) and add any required sections you are missing.

Reply:
Updated readme file according to the provided template in [this commit](https://github.com/microsoft/PowerPlatformConnectors/pull/1337/commits/3ab58071b349a4d70afcde2be80e5b110bef7be5)